include/cose/configure.h

```diff
@@ -2,6 +2,8 @@
 //  Determine which cryptographic library we are going to be using
```

Comment:
As long as we are going to rename this file, let's change the name of the file to cose_configure.h so that it does not conflict as a commonly used name.  This was on my todo list.